modules/openy_features/openy_egym_iwatch/src/scss/openy_egym_iwatch.scss

```diff
@@ -312,6 +312,10 @@
   .pre-footer {
     margin: -50px;
   }
+
+  .clearfix::before, .clearfix::after {
+    display: none;
```

Comment:
@podarok we are disabling this ONLY at our page.

Reply:
ok